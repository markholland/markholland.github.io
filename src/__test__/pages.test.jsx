import * as React from 'react'

import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Page from '../pages/404'

test('404 page renders', () => {
  const wrapper = shallow(
    <Page />
  )
  expect(toJson(wrapper)).toMatchSnapshot();
})
