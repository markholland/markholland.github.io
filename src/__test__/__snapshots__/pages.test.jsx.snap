// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page renders 1`] = `
<div>
  <h1>
    NOT FOUND
  </h1>
  <p>
    You just hit a route that doesn't exist...
  </p>
</div>
`;
